<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionWidget</class>
 <widget class="QWidget" name="ConnectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>219</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="LocalAddressEntry" name="comboBoxLocalAddress">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="TryToggle" name="tryToggleListen" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="TextEntry" name="lineEditLocalPort">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
     <property name="placeholderText">
      <string>Listen port</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="TextEntry" name="lineEditTargetAddress">
     <property name="placeholderText">
      <string>Target address</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="TextEntry" name="lineEditTargetPort">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
     <property name="placeholderText">
      <string>Target port</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TryToggle</class>
   <extends>QWidget</extends>
   <header>widgets/TryToggle.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TextEntry</class>
   <extends>QLineEdit</extends>
   <header>widgets/TextEntry.hpp</header>
  </customwidget>
  <customwidget>
   <class>LocalAddressEntry</class>
   <extends>QComboBox</extends>
   <header>widgets/LocalAddressEntry.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
